packages/indexer/src/starknet/parser.ts

```diff
@@ -0,0 +1,50 @@
+import { FieldElement } from "./felt";
+
+interface EventFilterParser {
+  eventFilter(name: string): Filter;
+  // eventParser(name: string): ParsedEvent[];
+}
+
+interface Filter {
```

Comment:
This interface is not needed, you can use [`EventFilter` from the same package](https://github.com/apibara/typescript-sdk/blob/main/packages/indexer/src/starknet/filter.ts#L130).